### PR TITLE
MP-541 :sparkles: service contracts no longer in data

### PR DESCRIPTION
**Pull request by nessy**

### CODE REVIEW

File: specification/paths/ServiceContracts.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                      "type": "string",
+                      "example": "$API_HOST/regions?page[number]=2&page[size]=30"
+                    },
+                    "next": {
+                      "type": "string",
+                      "example": "$API_HOST/regions?page[number]=4&page[size]=30"
+                    },
+                    "last": {
+                      "type": "string",
+                      "example": "$API_HOST/regions?page[number]=13&page[size]=30"
```
Comment (M4tini) [inline specification/paths/ServiceContracts.json]: all links: `regions` -> `service-contracts`

File: specification/schemas/ServiceContractsRelationship.json [hunk 2]
```diff
@@ -1,38 +1,14 @@
...
-      }
-    },
     "links": {
       "readOnly": true,
       "type": "object",
       "properties": {
         "related": {
           "type": "string",
-          "example": "$API_HOST/services/d35a7388-5300-49f9-9a84-01ca549a7775/contracts"
+          "example": "$API_HOST/services-contracts?filter[carrier_contract]=2cb32706-5762-4b96-9212-327e6afaeef"
```
Comment (M4tini) [inline specification/schemas/ServiceContractsRelationship.json]: `services-contracts` -> `service-contracts`

File: specification/paths/ServiceContracts.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                  "required": [
+                    "self",
+                    "first",
+                    "last"
+                  ],
+                  "additionalProperties": false,
+                  "properties": {
+                    "self": {
+                      "type": "string",
+                      "example": "$API_HOST/regions?page[number]=3&page[size]=30"
```
Comment (Shodske) [inline specification/paths/ServiceContracts.json]: Adjust these to be the correct urls.

File: specification/schemas/ServiceContractsRelationship.json (hunk 2 above)
Comment (Shodske) [inline specification/schemas/ServiceContractsRelationship.json]: `service-contracts`, not `services-contracts`

File: specification/paths/ServiceContracts.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
+{
+  "get": {
+    "tags": [
+      "ServiceContracts"
+    ],
+    "summary": "Get all the service contracts.",
+    "description": "This endpoint retrieves all the service contracts.",
+    "parameters": [
+      {
+        "name": "filter[carrier_contracts]",
```
Comment (M4tini) [inline specification/paths/ServiceContracts.json]: `filter[carrier_contracts]` -> `filter[carrier_contract]`

**Pull request merged**